Automattic/wp-calypso
# Render feature gate when site is domain only site

When a user tries to visit a page other than domain
management page, for a domain only site, we should
render component that tells the user she/he needs to
upgrade to a full site

This pull request replaces: #10686

![screen shot 2017-01-23 at 11 57 56](https://cloud.githubusercontent.com/assets/326402/22199364/3f9bce6a-e163-11e6-872d-6a4f3cf5c47b.png)

 
#### Testing instructions
  
1. Run `git checkout add/domain-only-site-featuregate` and start your server, or open a [live branch](https://calypso.live/?branch=add/domain-only-site-featuregate)
2. Open some restricted page for a domain only site, for example click the 'Write Button' on the masterbar [`Post` page](http://calypso.localhost:3000/post/)
3. Check that you displayed with an empty content page that saying you need to upgrade as shown at the screenshot
  
 #### Reviews
